app/api/search/search.js

```diff
@@ -599,14 +597,7 @@ const buildQuery = async (query, language, user, resources) => {
   if (query.sort) {
     const sortingProp = allUniqueProps.find(p => `metadata.${p.name}` === query.sort);
     if (sortingProp && sortingProp.type === 'select') {
```

Comment:
Should this be limited to just type select? Wouldn't multiselect properties also take advantage of this?